src/api/v1/orders.py

```diff
@@ -282,13 +282,13 @@ async def set_payment_status(
 
     try:
         order_service.set_payment_status(order_id, payment_result.result)
-        queue_service.register_queue_item(QueueItem(id=order_id))
+        if payment_result.result == True:
```

Comment:
```suggestion
        if payment_result.result is True:
```
Em geral é usado dessa forma pra garantir que a variável tem o valor `True`, pode ser que tenha um valor diferente e ainda passe pela validação.
Ex.: `1 == True` é verdadeiro.

Não acho que precise alterar, mas só queria deixar claro esse comportamento